new-log-viewer/src/components/CustomListItem.tsx

```diff
@@ -0,0 +1,44 @@
+import React from "react";
```

Comment:
Sure. I was thinking this is reusable across all components but since we don't have the need yet, I agree moving it into the directory where it's referenced (the `SidebarContainer/PanelTabs` directory) would cause less confusions.